From 8f4604c071ef11f9d70a4cb5b1a201b74d7a99ca Mon Sep 17 00:00:00 2001
From: jaffersulaiman <jaffersulaiman.mh@sasken.com>
Date: Tue, 22 Mar 2016 19:04:15 +0530
Subject: [PATCH 06/16] Messaging App is crashing when storage memory is full

Instead of showing low memory warning popup, Messaging App is crashing
because of popup using unsupported style attribute. So it is handled
with new style

Bug: CYNGNOS-2060
Change-Id: I00db39acf73df36c1ccdd7de65b92a41351ced21
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 609c379..738bca7 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -427,7 +427,7 @@
         </activity>
 
         <activity android:name=".ui.SmsStorageLowWarningActivity"
-                  android:theme="@style/Invisible"
+                  android:theme="@style/InvisibleNoDisplay"
                   android:configChanges="orientation|screenSize|keyboardHidden" />
 
         <activity android:name=".ui.appsettings.ApnSettingsActivity"
-- 
2.17.1

